tests/functional/data/patients.py

```diff
@@ -15,6 +15,7 @@ class Patient:
 SELF = Patient(nhs_number='9912003071',
                update=Update(nhs_number='9912003071',
                              path='telecom/0'))
+SELF_1 = Patient(nhs_number='9472063845')
```

Comment:
Might be better to remove this one and use `SELF_WITH_RELATED_PERSON` as the `Patient` returned in `self_patient_with_a_related_person`.